tests/nosetests/pyanaconda_tests/module_nvdimm_test.py

```diff
@@ -145,6 +145,7 @@ def _check_ignored(self, expected_devices):
                 "namespace0.0": Mock(blockdev="pmem0", mode=blockdev.NVDIMMNamespaceMode.SECTOR),
                 "namespace1.0": Mock(blockdev="pmem1", mode=blockdev.NVDIMMNamespaceMode.SECTOR),
                 "namespace2.0": Mock(blockdev="pmem2", mode=blockdev.NVDIMMNamespaceMode.MEMORY),
+                "namespace3.0": Mock(blockdev=None, mode=blockdev.NVDIMMNamespaceMode.SECTOR),
```

Comment:
I know it's just a test, but changing the mode to `blockdev.NVDIMMNamespaceMode.DEVDAX`  would make this more realistic, sector mode namespaces will always have a blockdev.